cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-network-settings-step/UnmanagedClustersNetworkSettings.tsx

```diff
@@ -283,59 +278,59 @@ function UnmanagedClusterNetworkSettings(props: Partial<StepProps>) {
-                                        </option>
-                                    );
-                                })}
-                            </select>
-                        </CdsSelect>
-                    </div>
+            <div cds-layout="m-b:md">
+                <div cds-layout="horizontal gap:sm p-y:md wrap:none">
+                    <CdsInput layout="vertical" control-width="shrink" cds-layout="align:shrink">
+                        <label cds-layout="wrap:none">IP Address</label>
+                        <input
+                            cds-layout="p:none"
+                            {...register(UNMANAGED_CLUSTER_FIELDS.IP_ADDRESS)}
+                            placeholder="127.0.0.1"
+                            onChange={handleFieldChange}
+                            defaultValue={umcState[STORE_SECTION_FORM].IP_ADDRESS}
+                        ></input>
+                    </CdsInput>
+                    <CdsInput layout="vertical" control-width="shrink">
+                        <label>Node Port</label>
+                        <input
+                            cds-layout="p:none"
+                            {...register(UNMANAGED_CLUSTER_FIELDS.NODE_PORT_MAPPING)}
+                            placeholder="80"
+                            onChange={handleFieldChange}
+                            defaultValue={umcState[STORE_SECTION_FORM].NODE_PORT_MAPPING}
+                        ></input>
+                    </CdsInput>
+                    <CdsInput layout="vertical" control-width="shrink">
+                        <label>Host Port</label>
+                        <input
+                            cds-layout="p:none"
+                            {...register(UNMANAGED_CLUSTER_FIELDS.HOST_PORT_MAPPING)}
+                            placeholder="80"
+                            onChange={handleFieldChange}
+                            defaultValue={umcState[STORE_SECTION_FORM].HOST_PORT_MAPPING}
+                        ></input>
+                    </CdsInput>
+                    <CdsSelect layout="vertical" control-width="shrink" onChange={handleProtocolChange}>
+                        <label>Protocol</label>
+                        <select cds-layout="p:none">
+                            {unmanagedClusterProtocol.map((unmanagedClusterProtocol, index) => {
+                                return (
+                                    <option cds-layout="m:md m-l:none" key={index}>
+                                        <label>{unmanagedClusterProtocol.label}</label>
+                                    </option>
+                                );
+                            })}
+                        </select>
+                    </CdsSelect>
+                </div>
+                <CdsControlMessage>
                     {combineNodeToHostPortMapping(
                         umcState[STORE_SECTION_FORM].IP_ADDRESS,
```

Comment:
You could also use the field names here: `umcState[STORE_SECTION_FORM][UNMANAGED_CLUSTER_FIELDS.IP_ADDRESS]`

Reply:
Good catch!